scripts/ci/lib.sh

```diff
@@ -1299,6 +1299,39 @@ __EOM__
       slack_mention="_unable to resolve Slack user for GitHub login ${author_login}_"
     fi
 
+    info "Converting junit failures to slack attachments"
+
+    local slack_attachments='
+[
+  {
+    "color": "#bb2124",
+    "blocks": [
+      {
+        "type": "section",
+        "text": {
+          "type": "plain_text",
+          "text": "Could not parse junit files. Check build logs for more information. 
+        }
+      }
+    ]
+  }
+]
+'
+    if [[ -n "${ARTIFACT_DIR}" ]]; then
+        if ! command -v junit-parse >/dev/null 2>&1; then
+            get_junit_parse_cli || exitstatus="$?"
+            if [[ "$exitstatus" == "0" ]]; then
```

Comment:
I think to handle the case when junit-parse is already in the test image:
```suggestion
            get_junit_parse_cli || true
        fi
        if command -v junit-parse >/dev/null 2>&1; then
```
We'll make a best effort try to get junit-parse and also handle when it is already present and not exit/error in either case.